content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/tutorials/pmc-opta-temp-ctrl/content.md

```diff
@@ -43,9 +43,9 @@ The system will send the temperature values from the Portenta Machine Control to
 ### Hardware Requirements
 
 - [Portenta Machine Control](https://store.arduino.cc/products/arduino-portenta-machine-control) (x1)
-- [Opta™ with RS-485](https://store.arduino.cc/collections/home-industrial-automation) (x1)
-- Micro-USB cable (x1)
-- USB-C® cable (x1)
+- Opta™ with RS-485: [Opta™ RS485](https://store.arduino.cc/products/opta-rs485), or [Opta™ WiFi](https://store.arduino.cc/products/opta-wifi) (x1)
```

Comment:
Just a minor suggestion to follow consistency across the documentation regarding possible changes:

```suggestion
Opta™ PLC with RS-485 support: [Opta™ RS485](https://store.arduino.cc/products/opta-rs485), or [Opta™ WiFi](https://store.arduino.cc/products/opta-wifi) (x1)
```

Reply:
added